package/submariner-route-agent.sh

```diff
@@ -9,23 +9,33 @@ else
     DEBUG="-v=4"
 fi
 
+function verify_iptables_on_host() {
+    chroot /host test -x /usr/sbin/$1 && { echo "0"; return; }
+    chroot /host test -x /sbin/$1 && { echo "1"; return; }
```

Comment:
Instead of hard-coding return-value-to-paths mappings, why not “return” the path directly?

```
    for file in {/usr,}/sbin/$1; do if [ -x $file ]; then echo $file; return; fi; done
    echo ""
```

then

```
iptables=$(verify_iptables_on_host)

if [ -n "$iptables" ]; then
    ...
else
    # Oops, no iptables
fi

Reply:
Depending on the location of the iptables/iptables-save utility on the host, we have to copy one of the wrapper files (i.e., L30 and L32 below). If we are just checking for [ -n "$iptables" ], how can we distinguish which file to copy?

Reply:
Ah, sorry, yes, I missed that.

Reply:
No problem @skitt. I missed a small change and realized that i pushed a wrong file. Fixed it now. Thanks for reviewing.